packages/wrangler/src/metrics/metrics-dispatcher.ts

```diff
@@ -43,13 +48,28 @@ export function getMetricsDispatcher(options: MetricsConfigOptions) {
 			});
 		},
 
+		/**
+		 * Dispatches `wrangler command started / completed / errored` events
+		 *
+		 * This happens on every command execution, and will (hopefully) replace sendEvent soon.
+		 * However to prevent disruption, we're adding under `sendNewEvent` for now.
+		 */
 		async sendNewEvent<EventName extends Events["name"]>(
 			name: EventName,
 			properties: Omit<
 				Extract<Events, { name: EventName }>["properties"],
 				keyof CommonEventProperties
 			>
 		): Promise<void> {
+			if (
+				properties.command === "wrangler telemetry disable" ||
+				properties.command === "wrangler metrics disable"
+			) {
+				return;
+			}
+			if (name === "wrangler command started") {
+				printMetricsBanner();
+			}
```

Comment:
I think we can remove the condition as the current logic should be safe to make sure it print once only. This also avoids the assumption that `wrangler command started` event to always be the first event.

Reply:
ce38f17b04457a39cf03a48ec53fbcfd2d8b3e40 sounds good 👍